guides/common/modules/proc_installing-and-configuring-puppet-agent-during-host-registration.adoc

```diff
@@ -47,10 +47,14 @@ endif::[]
 Alternatively, you can navigate to *Configure* > *Host Groups* and edit or create a host group to add host parameters only to a host group.
 . Enable the Puppet agent using a host parameter in global parameters or a host group.
 ifdef::katello,orcharhino,satellite[]
-Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
++
+* To use Puppet 8, add a host parameter named `enable-puppet8`, select the *boolean* type, and set the value to `true`.
+* To use Puppet 7, add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
```

Comment:
I need to test this in Satellite.

Reply:
Any updates @Lennonka ?

Reply:
Sorry, forgot about that. On it right now

Reply:
So, `enable-puppet7` works fine. On Satellite, we don't have Puppet agent 8 yet and it will be provided at a later time. So I suggest removing the puppet8 line for now.

Reply:
OK, made it non-satellite for now.